_maps/map_files/NSS-Sunset/NSS-Sunset.dmm

```diff
@@ -28414,6 +28414,7 @@
 /area/security/checkpoint/customs)
 "beT" = (
 /obj/machinery/quantumpad{
+	circuit = null;
```

Comment:
holy shit what are you doing
go back to transit pipes this is not okay

Reply:
no, transit pipes are ugly

Reply:
you had to remove a possible advantage smart players could get by snowflaking it in the map file do you realize how much of a bad design decision this is